client/lib/tracks.js

```diff
@@ -17,7 +17,7 @@ const tracksDebug = debug( 'wc-admin:tracks' );
  */
 
 export function recordEvent( eventName, eventProperties ) {
-	tracksDebug( 'recordevent %s %o', eventName, eventProperties );
+	tracksDebug( 'recordevent %s %o', 'wc_admin_' + eventName, eventProperties );
```

Comment:
Can you fix the comment in the doc block above?

Reply:
good find, updated.